app/templates/views/service-settings/edit-service-billing-details.html

```diff
@@ -0,0 +1,35 @@
+
+{% extends "withnav_template.html" %}
+{% from "components/page-header.html" import page_header %}
+{% from "components/page-footer.html" import page_footer %}
+{% from "components/form.html" import form_wrapper %}
+
+{% from "components/textbox.html" import textbox %}
+
+{% block service_page_title %}
+  Change billing details
+{% endblock %}
+
+{% block maincolumn_content %}
+
+  {{ page_header(
+    'Change billing details',
+    back_link=url_for('main.service_settings', service_id=current_service.id)
+  ) }}
+
+  {% call form_wrapper() %}
+    {{ form.billing_contact_names(param_extensions={"classes": "govuk-!-width-full"}) }}
+    {{ form.billing_contact_email_addresses(param_extensions={"classes": "govuk-!-width-full"}) }}
+    {{ form.billing_reference(param_extensions={"classes": "govuk-!-width-full"}) }}
+    {{ form.purchase_order_number(param_extensions={"classes": "govuk-!-width-full"}) }}
```

Comment:
You can make these long numbers a bit more readable with some extra letter spacing (see https://github.com/alphagov/notifications-admin/pull/1545 for screenshots)
```suggestion
  <div class="extra-tracking">
      {{ form.billing_reference(param_extensions={"classes": "govuk-!-width-full"}) }}
      {{ form.purchase_order_number(param_extensions={"classes": "govuk-!-width-full"}) }}
  </div>
```

Reply:
Ohh, cool! Why is it call tracking, not spacing, tho? 🤔

Reply:
They both mean the same thing. Maybe we should change it to `extra-letter-spacing` to be more self explanatory for people not familiar with typography terminology.

Reply:
Hmmm... that's a good idea, `extra-letter-spacing` sounds self-explanatory :3

Reply:
I can add it in my org billing details PR as a separate commit if you want?

Reply:
Let’s do a separate PR for that

Reply:
Sounds good 👍🏼